code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -66,8 +66,7 @@
 		if(MODE_RAD)
 			show_rads()
 
-/obj/item/clothing/glasses/meson/engine/proc/show_rads()
-	var/mob/living/carbon/human/user = loc
+/mob/proc/show_rads(range)
```

Comment:
Not the best location for a `mob` level proc. Perhaps `mob.dm`?

Reply:
Addressed in 000b89a